Calibration/EcalCalibAlgos/interface/EcalEleCalibLooper.h

```diff
@@ -60,64 +62,64 @@ class EcalEleCalibLooper : public edm::EDLooper {
 
 private:
   //! EcalBarrel Input Collection name
-  edm::InputTag m_barrelAlCa;
+  const edm::InputTag m_barrelAlCa;
   //! EcalEndcap Input Collection name
-  edm::InputTag m_endcapAlCa;
+  const edm::InputTag m_endcapAlCa;
+  //! To take the electrons
+  edm::InputTag m_ElectronLabel;
 
   //! reconstruction window size
-  int m_recoWindowSidex;
-  int m_recoWindowSidey;
+  const int m_recoWindowSidex;
+  const int m_recoWindowSidey;
 
   //! eta size of the sub-matrix
-  int m_etaWidth;  //PG sub matrix size and borders
+  const int m_etaWidth;  //PG sub matrix size and borders
   //! eta size of the additive border to the sub-matrix
   //    int m_etaBorder ; //FIXME
   //! phi size of the sub-matrix
-  int m_phiWidthEB;
+  const int m_phiWidthEB;
   //! phi size of the additive border to the sub-matrix
   //    int m_phiBorderEB //FIXME;
```

Comment:
is this commented out code expected to be used?

Reply:
I do not know. I just did the esConsumes migration and I do not use this code otherwise.